wire-ios/Wire-iOS/Sources/UserInterface/StartUI/StartUI/StartUIViewController.swift

```diff
@@ -108,7 +108,17 @@ final class StartUIViewController: UIViewController {
     }
 
     var showsGroupSelector: Bool {
-        isAppsFeatureEnabled && SearchGroup.all.count > 1 && userSession.selfUser.canSeeServices
+
+        // restore old behavior until `apps` feature flows are complete
+        #if true
+            return SearchGroup.all.count > 1 &&
+                userSession.selfUser.canSeeServices &&
+                userSession.defaultProtocol != .mls
+        #else
+            // TODO: [WPB-21834] consider adding a client-side feature flag for the new behavior
+            return isAppsFeatureEnabled && SearchGroup.all.count > 1 && userSession.selfUser.canSeeServices
+        #endif
```

Comment:
question: could you do a OR which would work for both cases? 


```suggestion
        // restore old behavior until `apps` feature flows are complete
        
            return (!isAppsFeatureEnabled && SearchGroup.all.count > 1 &&
                userSession.selfUser.canSeeServices &&
                userSession.defaultProtocol != .mls
                ) || (isAppsFeatureEnabled && SearchGroup.all.count > 1 && userSession.selfUser.canSeeServices)
```

Reply:
Wouldn't that enable apps even for teams who shouldn't have it enabled (paid feature)?

Reply:
right